packages/algob/src/internal/cli/project-creation.ts

```diff
@@ -57,7 +52,12 @@ function copy (directory: string, location: string): void {
 function copySampleProject (location: string, isTSProject: boolean): void {
   const packageRoot = getPackageRoot();
   const sampleProjDir = path.join(packageRoot, "sample-project");
-
+  console.log("Location", location);
+  if (fsExtra.pathExistsSync(`./${location}/algob.config.js`)) {
+    throw new BuilderError(ERRORS.GENERAL.INIT_INSIDE_PROJECT, {
+      clashingFile: location
+    });
+  }
   console.log(chalk.greenBright("Initializing new workspace in " + process.cwd() + "."));
```

Comment:
shouldn't this be changed to `./${location}`, in case we're initializing a workspace in a new directory? Could you please check?

Reply:
done